src/index.js

```diff
@@ -319,6 +320,17 @@ async function migrate({app, path: dir, projectId, dryrun, debug = false, requir
     auth: {getClient: () =>targetClient},
   });
 
+	const [elasticsearchCredentials] = await secretManager.accessSecretVersion({
+		name: `projects/${projectId}/secrets/AKENEO_ELASTICSEARCH_CREDENTIALS/versions/latest`,
```

Comment:
maybe we should add a check if we are targeting local env or not? We could aslodo a try catch on the secret manager fetch to fail only if we are targeting gcp?

Reply:
Thanks, totally forgot about local. I updated the code so that we target localhost:9200 when projectId is `local`. In this case, we don't need to use the secret manager :slightly_smiling_face:

Reply:
:dab: